broadinstitute/pearl
# [JN-417] Implement basic kit management UI

Also adds random kit requests to the bulk test user creation feature.

To test:
1. Wipe and repopulate your database (or try without, but there might be issues with previously-populated data... which I don't like so I'd actually like to know if you're able to test this without wiping).
2. Start the admin UI and go to https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/kits
3. Play around with the filtering, sorting, and column visibility.
4. Click the "Kits" tab to see the one default populated kit request.
5. Click back over to the "Enrollees" tab, select the `OHSALK` row, click "Send sample collection kit", and click the "Request Kit" button.
6. Click back over to the "Kits" tab to see that there are now 2 kit requests for `OHSALK`.
7. In another window, go to http://localhost:8080/swagger-ui.html#/populate/bulkPopulateEnrollees and fill in `ourhealth`/`sandbox`/`ourheart`/`20` to generate 20 random users.
8. Back in the admin UI, click back over to the "Enrollees" tab to see the generated enrollees. All should have the basics and cardiometabolic medical history surveys completed. A few may oddly not be consented (5% chance). Many will have an existing kit request (80% chance).
9. Click back over to the "Kits" tab to see the generated kits. Most will be "Prepared", fewer will be "Sent", and fewer still will be "Returned".
10. Try out the status filter.

![image](https://github.com/broadinstitute/juniper/assets/19228696/63a696b1-ce0a-41cf-9650-30049ed4d399)
![image](https://github.com/broadinstitute/juniper/assets/19228696/0e880500-82e1-4f29-91d6-2441d136c5c1)
![image](https://github.com/broadinstitute/juniper/assets/19228696/b2ae98c7-5f5f-46a1-93e5-b44ec8e7a14b)
